core/templates/dev/head/app.js

```diff
@@ -393,7 +393,9 @@ oppia.factory('urlService', ['$window', function($window) {
       return params;
     },
     isIframed: function() {
-      return !!(this.getUrlParams().iframed);
+      var pathname = this.getPathname();
+      var urlParts = pathname.split('/');
+      return urlParts.indexOf('embed') !== -1;
```

Comment:
There's going to be a specific segment of the URL that contains 'embed'. Can we just check that segment, rather than doing a search?